### DISCUSSION TITLE
Brainstorming

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/sys-581-group-project#1] [Ideas]
Project: Command, Control and Communications (C 3 ) System
Function: Provide and maintain the all internal and external (e.g., Earth) Colony Command, Control and Communications
required for healthy, safe and sustainable human habitat consistent with the Colony establishment timeline below. _No timeline provided_

**Comment by dyc3** [comment]
Some general requirements:
- Communication needs to be somewhat async. Lag for comms between earth and mars is ~20m, too slow for real time comms.
- Messages should have read receipts to be more reliable
- Should be scalable to accommodate long term population growth
- Should be scalable to accommodate new colonies elsewhere in the solar system
- Unmanned missions must be reliably automated, and remotely updatable.
- Communications must be encrypted to protect the privacy of personal communications, and the integrity of commands
- Because planet alignments, there are going to be periods where messages can't be sent. (https://mars.nasa.gov/news/8861/the-mars-relay-network-connects-us-to-nasas-martian-explorers/)

**Comment by nick-dimeglio** [comment]
- Long distance system health to remotely see performance stats on the colony from Earth
- Need to have near perfect reliability of being able to send messages to and from any given colony, and during travel

**Comment by juan0campo8** [comment]
- Should operate within the power generation constraints of the entire colony.

**Comment by nick-dimeglio** [comment]
Mars User Requirements:

- I should be able to send a message to Earth control at all times.
- I should be able to communicate with my family at home
- I should be able to control any remote and/or unmanned missions reliably
- 

Earth User Requirements:
- Constant communication with Martian colonists
-
